pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+# addopts for html have to go in test.sh because of packit (rpm specfile runs unit tests)
```

Comment:
@ben-alkov you should correct this comment.
Packit is just tooling that makes build for you, however, the problem here is that you have %check running pytest in specfile. So you could get the same problems when you will use mock to create a build.